pkg/curatedpackages/regional_registry.go

```diff
@@ -101,10 +101,10 @@ func TestRegistryWithAuthToken(authToken, registry string, do Do) error {
 
 // GetRegionalRegistry get the regional registry corresponding to defaultRegistry in a specific region.
 func GetRegionalRegistry(defaultRegistry, region string) string {
-	if strings.Contains(defaultRegistry, devAccount) {
+	if strings.Contains(defaultRegistry, devRegistry) {
```

Comment:
For this do we need to change the bundle to have the dev registry??

Reply:
ah yeah I think I need to have two separate vars for the old/new dev registries.